runtime/doc/version9.txt

```diff
@@ -195,6 +195,7 @@ New and extended functions: ~
 |readdirex()|		get a List of file information in a directory
 |reduce()|		reduce a List to a value
 |searchcount()|		get number of matches before/after the cursor
+|searchstr()|		get string that matches the pattern
```

Comment:
Can you update the `function-list` section in usr_41.txt to include this function?